ExampleService/ExampleService_WebApi/.vscode/tasks.json

```diff
@@ -0,0 +1,24 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
```

Comment:
Should the .vscode folder be in the top level `ExampleService` folder?

Reply:
Move to the top level.